src/themes/default/templates/elements/article_listing.html

```diff
@@ -34,18 +34,29 @@ <h6 class="card-subtitle mb-2 text-muted">{% for author in article.frozen_author
                         {% endif %}{% endif %}{% endfor %}
                 </h6>
                 <p class="card-text">
-                    <span class="date"><i class="fa fa-calendar"></i> {{ article.date_published|date:"Y-m-d" }}</span> 
-                    
+                    <span class="date"><i class="fa fa-calendar"></i> {{ article.date_published|date:"Y-m-d" }}</span>
                     {% if article.issue %}
-                    <i class="fa fa-book"></i> 
```

Comment:
Whitespace here makes the icon site off the text.

Reply:
As discussed, newlines are rendered as space, no need for this trailing space here.